src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in

```diff
@@ -13,8 +13,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="${MicrosoftAspNetCoreAllPackageVersion}" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="${MicrosoftAspNetCoreAllPackageVersion}" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="${MicrosoftEntityFrameworkCoreSqlitePackageVersion}" Condition=" '$(IndividualLocalAuth)' == 'True' AND '$(UseLocalDB)' != 'True'" />
```

Comment:
In our last discussion, we thought that Sqlite is only required for non-windows OS but I noticed that we use Sqlite when not using localDB regardless of which OS we are on: https://github.com/aspnet/templating/blob/release/2.1/src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs#L46. 

We either need to add the package reference to Sqlite or we need to modify Startup to not call UseSqlite on Windows.

Reply:
Spoke with @DamianEdwards, we should be adding this reference whenever localdb is not used including on Windows. So the changes here are correct.